### ISSUE TITLE
Highlighting desn't work after 0.2.6 update

### ISSUE THREAD

**Issue by EvgeniyPeshkov** [EvgeniyPeshkov/syntax-highlighter#10]
#### !!!Attention!!!
This release changes names of some syntax terms.
I apologize to all users, who has already tuned colors, for this inconvenience.
Please rename {Syntax Highlighter} colors in `settings.json` the following way:
* `"syntax.namespace" -> "syntax.scope"`
* `"syntax.keyword_constant" -> "syntax.constant"`
* `"syntax.keyword_directive" -> "syntax.directive"`
* `"syntax.keyword_control" -> "syntax.control"`
* `"syntax.keyword_operator" -> "syntax.operator"`
* `"syntax.storage_modifier" -> "syntax.modifier"`

Basically, this is not an issue. I've posted it myself to warn about these changes.
I will close it in several days when most users update their settings.

**Comment by EvgeniyPeshkov** [comment]
As a week has passed, I'll close this issue.
